static/js/addons/cl.utils.js

```diff
@@ -95,8 +100,29 @@ var Cl = window.Cl || {};
          * @param token {String} namespace
          */
         getStorage: function (token) {
-            if (typeof (Storage) !== void(0)) {
+            if (token && this._checkStorage()) {
                 return localStorage.getItem(token);
+            } else {
+                return false;
+            }
+        },
+
+        /**
+         * Localstorage shim from Modernizr
+         *
+         * @method _checkStorage
+         * @private
+         */
+        _checkStorage: function () {
```

Comment:
does it make sense to make this self execute so it's only executed once when the page load and not every time we try to access storage?